its/ruling/src/integrationTest/resources/expected/kubernetes/kubernetes-S2260.json

```diff
@@ -17,12 +17,33 @@
 "kubernetes-project:sources/kubernetes/examples/staging/storage/vitess/vttablet-pod-template.yaml": [
 0
 ],
+"kubernetes-project:sources/kubernetes/helm-chart-sonarqube/charts/sonarqube-dce/templates/config.yaml": [
```

Comment:
Those new files with parsing errors are ConfigMaps that were not being detected before

Reply:
OK and please check why they produce parsing error - something new or known bug

Reply:
Not sure if it is known as a part of your ticket about blank lines:
```
data: #29
  sonar.properties: | #30 #33 #36
 #37
```
leads to
```
13:28:10.791 DEBUG: while scanning a block scalar
 in reader, line 12, column 21:
      sonar.properties: | #12 #15 #18
                        ^
 the leading empty lines contain more spaces (3) than the first non-empty line.
 in reader, line 13, column 1:
    ---
    ^
```

Reply:
```yaml
data: #10
  install_plugins.sh: |- #11 #18
 #19
```
=>
```
 in reader, line 11, column 23:
      install_plugins.sh: |- #11 #18
                          ^
 the leading empty lines contain more spaces (3) than the first non-empty line.
 in reader, line 12, column 2:
     #19
     ^
```

Reply:
It's snakeyaml issue and it's known: https://sonarsource.atlassian.net/browse/SONARIAC-751

Reply:
Another one, I believe, is related to a part like this:
```yaml
--- #30
 #31 #32
---
```
Maybe we should filter out `---` with comments not only before Helm evaluation but after as well?